scripts/digest_inputs

```diff
@@ -32,6 +32,19 @@ DEV_TAGS="${DEV_TAGS:-}"
 DOCKERFILE="${DOCKERFILE:-Dockerfile}"
 TAGS="${TAGS:-}"
 REDHAT_TAG="${REDHAT_TAG:-}"
+PUSH_AUTO_DEV_TAGS="${PUSH_AUTO_DEV_TAGS:=false}"
```

Comment:
Do we need to allow passing this in [somewhere around here in the action.yaml file](https://github.com/hashicorp/actions-docker-build/blob/main/action.yml#L115-L120)?